packages/suite-desktop/src/libs/processes/TorProcess.ts

```diff
@@ -32,23 +39,12 @@ export class TorProcess extends BaseProcess {
         });
     }
 
-    async status(): Promise<Status> {
-        try {
-            const isAlive = await this.torController.status();
-            if (isAlive) {
-                return {
-                    service: true,
-                    process: true,
-                };
-            }
-        } catch {
-            //
-        }
-
-        // process
+    async status(): Promise<TorProcessStatus> {
+        const torControllerStatus: TorControllerStatus = await this.torController.getStatus();
         return {
-            service: false,
+            service: torControllerStatus === TOR_CONTROLLER_STATUS.CircuitEstablished,
```

Comment:
is `service` good name?

Reply:
I agree is not the best, I got confused first time I saw it but it is defined by BaseProcess and it is used in all the Processes we use in suite https://github.com/trezor/trezor-suite/blob/1630a9ce65a207ab13efa9acee7288df3794f1ba/packages/suite-desktop/src/libs/processes/BaseProcess.ts#L71